modules/hcp-ibmpower-scale-np.adoc

```diff
@@ -0,0 +1,124 @@
+----
++
+.Example output
+[source,terminal]
+----
+BMH: Agent: 50c23cda-cedc-9bbd-bcf1-9b3a5c75804d State: known-unbound
+BMH: Agent: 5e498cd3-542c-e54f-0c58-ed43e28b568a State: insufficient
+----
+
+. Run the following command to see the transition phases:
++
+[source,terminal]
+----
+$ oc -n <hosted_control_plane_namespace> get agent
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                   CLUSTER           APPROVED       ROLE        STAGE
+50c23cda-cedc-9bbd-bcf1-9b3a5c75804d   hosted-forwarder   true          auto-assign
+5e498cd3-542c-e54f-0c58-ed43e28b568a                      true          auto-assign
+da503cf1-a347-44f2-875c-4960ddb04091   hosted-forwarder   true          auto-assign
+----
+
+. Run the following command to generate the `kubeconfig` file to access the hosted cluster:
++
+[source,terminal]
+----
+$ hcp create kubeconfig --namespace <hosted_cluster_namespace> --name <hosted_cluster_name> > <hosted_cluster_name>.kubeconfig
+----
+
+. After the agents reach the `added-to-existing-cluster` state, verify that you can see the {product-title} nodes by entering the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <hosted_cluster_name>.kubeconfig get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                             STATUS   ROLES    AGE      VERSION
+worker-zvm-0.hostedn.example.com Ready    worker   5m41s    v1.24.0+3882f8f
+worker-zvm-1.hostedn.example.com Ready    worker   6m3s     v1.24.0+3882f8f
+----
+
+. Enter the following command to verify that two machines were created when you scaled up the `NodePool` object:
++
+[source,terminal]
+----
+$ oc -n <hosted_control_plane_namespace> get machine.cluster.x-k8s.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                CLUSTER  NODENAME PROVIDERID     PHASE     AGE   VERSION
+hosted-forwarder-79558597ff-5tbqp   hosted-forwarder-crqq5   worker-zvm-0.hostedn.example.com   agent://50c23cda-cedc-9bbd-bcf1-9b3a5c75804d   Running   41h   4.15.0
+hosted-forwarder-79558597ff-lfjfk   hosted-forwarder-crqq5   worker-zvm-1.hostedn.example.com   agent://5e498cd3-542c-e54f-0c58-ed43e28b568a   Running   41h   4.15.0
+----
+
+. Run the following command to check the cluster version:
++
+[source,terminal]
+----
+$ oc --kubeconfig <hosted_cluster_name>.kubeconfig get clusterversion
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                         VERSION       AVAILABLE   PROGRESSING   SINCE   STATUS
+clusterversion.config.openshift.io/version   4.15.0        True        False         40h     Cluster version is 4.15.0
+----
+
+. Run the following command to check the Cluster Operator status:
++
+[source,terminal]
+----
+$ oc --kubeconfig <hosted_cluster_name>.kubeconfig get clusteroperators
+----
++
+For each component of your cluster, the output shows the following Cluster Operator statuses:
```

Comment:
This bulleted list is not formatted correctly in the output. I think you just need to add a line break before the first bullet point.

Reply:
Fixed